test/release/examples/benchmarks/shootout/README

```diff
@@ -31,7 +31,7 @@ repository under the test/studies/shootout/ directory.
 
 The provided Makefile can be used to compile the programs, or they can
 be run in correctness or performance modes using the Chapel testing
-system (see $CHPL_HOME/util/sub_test).
+system (see $CHPL_HOME/util/start_test).
```

Comment:
While you're right that the original intention was to point the reader at start_test, start_test no longer contains the instructions for how to run a particular job.  With that in mind, it's probably more useful to point the reader at $CHPL_HOME/doc/developer/bestPractices/TestSystem.rst (being sure to note that this file is only available in the github repository and not the release tarball)